elements/howto-menu/demo.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    display: block;
```

Comment:
With the implicit `position: static` the menu will push down following content when opened and re-layout the entire page. I think `position: absolute` & some JavaScript to set the position might be worth a try.